secedgar/cik_lookup.py

```diff
@@ -31,6 +51,21 @@ def __init__(self, lookups, client=None, **kwargs):
             self._lookups = lookups
         self._params = {'action': 'getcompany'}
         self._client = client if client is not None else NetworkClient(**kwargs)
+        self._lookup_dict = None
+        self._ciks = None
+
+    @property
+    def ciks(self):
```

Comment:
where is `.ciks` used?

Reply:
This is more of a convenience function for users to easily see what CIKs are being looked up.

Reply:
```python3
        if self._ciks is None:
            self._ciks = list(self.lookup_dict.values())
        return self._ciks
```